Code-the-Dream-School/dd-prac-team1-front
# Navbar PRA-89, logout PRA-91, username in the session storage PRA-102

## Navbar after Login - Logout to Home page

## Requirements

1. PRA-89  Add navbar with search, add to favorites, logout buttons with appropriate routes
3. PRA-91
 1) The logout request is sent to backend successfully

2) Logout happens after clicking the logout text in Navbar

3) The user gets redirected to Home page

4) The username is stored in the session storage 

## Notes



## Test Steps

1) Log in and see the navbar. 
2) Log out and check in the console that the logout has been successful. 
3) After the successful logout you are navigated to the Home page.
4) You can also check the mobile version with the hamburger menu.
5) You can see the user's name in the session storage when they are logged in.
## UI/UX Outcome

<img width="1471" alt="Screen Shot 2023-08-04 at 16 06 50" src="https://github.com/Code-the-Dream-School/dd-prac-team1-front/assets/114008959/6e79c41a-8647-4541-959c-40c5e2e7acb8">

<img width="549" alt="Screen Shot 2023-08-04 at 16 12 27" src="https://github.com/Code-the-Dream-School/dd-prac-team1-front/assets/114008959/ee371b02-b06e-429c-8f1e-bdb4ecf03cac">

<img width="562" alt="Screen Shot 2023-08-04 at 16 10 43" src="https://github.com/Code-the-Dream-School/dd-prac-team1-front/assets/114008959/5b28f1e6-c86e-4a52-a185-69aceb7f168f">

<img width="803" alt="Screen Shot 2023-08-04 at 16 09 19" src="https://github.com/Code-the-Dream-School/dd-prac-team1-front/assets/114008959/fd6a93f5-adb4-4399-b602-de135dbea281">

## Checklist

- The Linear ID is in the PR title
- My code follows best practices
- Documentation is included where needed
